src/routes/accommodationRoute.js

```diff
@@ -81,16 +82,18 @@ const fUpload = fileUpload({
 });
 
 const {
-  isSupplierAccommodation, checkForImages, checkForImagesUpdate
+  isSupplierAccommodation, checkForImages, checkForImagesUpdate,
 } = accMidd;
 const { checkToken } = userMidd;
 const {
-  addAccommodation, getAccommodation, deleteAccommodation, editAccommodation
-} = accommodationCont;
+  addAccommodation, getAccommodation, deleteAccommodation, editAccommodation,
+} = accommodationController;
 
 app.patch('/:id', fUpload, checkToken, roles.checkHost, isSupplierAccommodation, checkForImagesUpdate, valid.editAccommodation, editAccommodation);
 app.delete('/:id', checkToken, roles.checkHost, isSupplierAccommodation, deleteAccommodation);
 app.post('/', fUpload, checkToken, roles.checkHost, valid.accommodation, checkForImages, addAccommodation);
 app.get('/', checkToken, accMidd.checkViewAccommodation, getAccommodation);
+app.get('/:id/rating', bookingsController.getAverageRating);
+app.get('/:id/feedbacks', bookingsController.getAccommodationFeedbacks);
```

Comment:
you didn't handle that error, when someone provide an id like this `#@123` (there is no validator)

Reply:
I'm correctly checking if the provided id is a number or not. and eventually returning an error message

Reply:
You checked on one `route/:id/review`

Reply:
Done!